AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -605,7 +635,7 @@ - (ASSizeRange)dataController:(ASDataController *)dataController constrainedSize
   }
   
   if (_asyncDelegateImplementsInsetSection) {
-    sectionInset = [_asyncDelegate collectionView:self layout:self.collectionViewLayout insetForSectionAtIndex:indexPath.section];
+    sectionInset = [(id<ASCollectionViewDelegateFlowLayout>)_asyncDelegate collectionView:self layout:self.collectionViewLayout insetForSectionAtIndex:indexPath.section];
```

Comment:
Missed any related stuff to this, why are we casting to FlowLayout here?  Presumably it is not always a flow layout, so is there different protocol naming / structure that would fix?